src/modules/reporting/components/reporting-closed/reporting-closed.jsx

```diff
@@ -50,6 +54,9 @@ export default class ReportingClosed extends Component {
           linkType={TYPE_CLOSED}
           paginationPageParam="reporting-closed-page"
           isMobile={p.isMobile}
+          isForking={p.isForking}
+          isForkingMarketFinalized={p.isForkingMarketFinalized}
+          forkingMarket={p.forkingMarket}
         />
       </section>
```

Comment:
changes not needed Market properties will pick up properties from component container